unity-sdk/AndroidPlugin/OptimoveUnityPlugin/build.gradle

```diff
@@ -27,10 +27,15 @@ android {
 
 dependencies {
 
+    // Add library dependency to retrieve FCM push tokens
+    implementation platform('com.google.firebase:firebase-bom:28.2.0')
+    implementation 'com.google.firebase:firebase-messaging'
     implementation 'androidx.appcompat:appcompat:1.5.1'
     implementation 'com.google.android.material:material:1.6.1'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     implementation 'com.optimove.android:optimove-android:6.+'
```

Comment:
dependencies on firebase and optimove here should be compile only. This lets customers decide which versions they want and avoid conflicts. 

If we want fully functional AndroidPlugin project -- we would add implementation dependencies to the app module